packages/react/src/components/header/components/headerLanguageSelector/HeaderLanguageSelector.tsx

```diff
@@ -38,6 +48,17 @@ const renderLanguageNode = (language: LanguageOption) => {
   return <LanguageButton key={language.value} value={language.value} label={language.label} />;
 };
 
+const defaultLanguageSorter: LanguageSelectorProps['sortLanguageOptions'] = (options, selectedLanguage) => {
+  const hasPrimaryLanguages = options.findIndex((option) => typeof option.isPrimary !== 'undefined') > -1;
```

Comment:
Suggestion:
`const hasPrimaryLanguages = options.some((option) => typeof option.isPrimary !== 'undefined');`

Reply:
This is good 🤌